content/docs/command-reference/repro.md

```diff
@@ -7,9 +7,9 @@ correct order.
 ## Synopsis
 
 ```usage
-usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [--dry] [-i]
-                 [-p] [-P] [-R] [--no-run-cache] [--force-downstream]
-                 [--no-commit] [--downstream] [--pull] [--glob]
+usage: dvc repro [-h] [-q | -v] [-f] [-s] [-m] [-i] [-p] [-P] [-R]
```

Comment:
So this groups all the short flags at the start? Looks good to me.

Reply:
Yeah I think that on the core repo, currently new arguments are just added at the bottom so there's no logical order.

My criteria here isn't super defined but broadly: put general ones in the first line, esp. if they're shot flags; Then group by functionality in the order in which we think people will use them most -- in the case above (`repro`) realted to deps/params, then outs/metrics/plots, then DVCLive.

Reply:
If we move `[--no-exec] [--no-run-cache] [--no-commit]` to the bottom for `run` and `stage add`, can we put the related `repro` options `[--pull] [--no-run-cache] [--no-commit]` at the end here?

Reply:
Should the criteria be translated back to dvc core repo?

Reply:
Yeah, once there is an agreement on the order of everything, we should update the core repo.

Reply:
> move [--no-exec] [--no-run-cache] [--no-commit] to the bottom for run

Done. Well, I moved them above the DVCLive ones, PTAL

> and stage add

Doesn't have them.

> put the related repro options [--pull] [--no-run-cache] [--no-commit] at the end

Done.

---

- [ ] BTW `run --no-exec` is equivalent to `repro --dry` right? May want to rename the former to "dry" as well.

Reply:
`repro` is not read-only.